### ISSUE TITLE
Adjust the scale of interface elements of Placeables such that they look the same regardless of the configured grid size

### ISSUE THREAD

**Issue by LukasPrism** [foundryvtt/foundryvtt#10608]
## Implemented Changes

- All canvas UI elements which size is not configured by the users in (true) pixels are now adjusted based on the grid size (i.e. proportional to the grid size). Additionally all those elements are scaled with the *Interface Scale*.
- Added `canvas.dimension.scale.min`, which is the minimum scale of the canvas stage (zoom level).
- Added `canvas.dimension.scale.max`, which is the maximum scale of the canvas stage (zoom level).
- Added `canvas.dimension.scale.default`, which is the default scale of the canvas stage (zoom level).
- Added `canvas.dimension.uiScale`, which is the multiplier that converts normalized pixels to true pixels. 100 normalized pixels are equal to the grid size if the *Interface Scale* is set to 1.0.

## Original Issue

Please consider having nameplates, ruler UI and other canvas elements (like door icons) scale based on grid width like the token HUD rather than 'Grid Size' (pixel density). That would alleviate the UI scaling wildly between, say, a 200px map and a 70px map (both fairly common scenarios). 

Folk being able to set a scale factor that they prefer would be ideal, which would also help with accessibility. 

Other known elements that fall into this category are: border width, grid line width, and the edge falloff distance of light/vision sources (thanks Eber).

**Comment by aaclayton** [comment]
I don't understand what you mean as the difference between "grid width" (not a defined concept) and "grid size". For a square grid, for example, the width of a grid space *is* the "size" of that grid space. 

I understand the spirit of the suggestion that there are certain elements which scale helpfully or un-helpfully based on the grid size but I would appreciate if you could make the nature of the issue more clear. Perhaps side-by-side screenshots to illustrate exactly what you are asking for.

**Comment by LukasPrism** [comment]
Yeah sorry not an easy one to explain. I come from the graphic design world so it’s usually a pixels per inch thing. Grid Width as in the size of a grid square (medium token) irrespective of what you call ‘Grid Size’ which is the pixel density. So, for instance the point size of a nameplate would be a percentage of the grid width or height and not based on the number of pixels in said grid square.

**Comment by LukasPrism** [comment]
I understand one way to circumvent this would be to resize the map itself, however maps like these Dungeon of the Mad Mage maps by Cyren are 70px for the reason that they're very expansive in terms of number of grid squares. The pixel size of the 70px map above is already 12040 x 16520.

**Comment by dev7355608** [comment]
Related issues:
- https://github.com/foundryvtt/foundryvtt/issues/4916
- https://github.com/foundryvtt/foundryvtt/issues/4920
- https://github.com/foundryvtt/foundryvtt/issues/5116
- https://github.com/foundryvtt/foundryvtt/issues/10608

The min/max zoom level and the point source mesh edge falloff distance needs adjustments too.
